src/main/java/com/yurjinia/project_structure/board/service/BoardService.java

```diff
@@ -64,11 +60,21 @@ public BoardEntity getBoard(String boardCode, String projectCode) {
         return boardRepository.findByCodeAndProjectCode(boardCode, projectCode).orElseThrow(() -> new CommonException(ErrorCode.BOARD_NOT_FOUND, HttpStatus.NOT_FOUND));
     }
 
+    public void save(BoardEntity boardEntity) {
+        boardRepository.save(boardEntity);
+    }
+
     private void validateIfBoardNotExist(String boardName, String boardCode, String projectCode) {
         if (boardRepository.existsByNameOrCodeAndProject(boardName, boardCode, projectCode)) {
             throw new CommonException(ErrorCode.BOARD_ALREADY_EXISTS, HttpStatus.CONFLICT,
                     List.of("Board by name " + boardName + " or code " + boardCode + " already exists"));
         }
     }
 
+    private void updateIfNotBlank(String newValue, Consumer<String> updater) {
```

Comment:
move this method to separate Utils class, check all occurrences in other classes.

name the class - `MetadataUtils`, and check the name of this method in TicketService.